etsin_finder/frontend/js/stores/view/datasetquery.js

```diff
@@ -62,6 +68,11 @@ class DatasetQuery {
     })
   }
 
+  @action
+  getFilesV2() {
```

Comment:
Assuming this.Files.openDataset(this.results) will fetch something and store results into a variable, the name of the action "getFilesV2" is misleading. "get" suggests that this function is merely a getter but it really does much of other stuff. fetchAndStore would be better in this case or just fetch if storing is just a byproduct of the function (and arg is optional).

Reply:
done